### DISCUSSION TITLE
How to add /dev/ to ttyusb0?

### DISCUSSION THREAD

**Discussion by landocarlesien** [aqualinkd/AqualinkD#358] [Q&A]
How do I add /dev/ to /TTYUSB0? I can see TTYUSB0 connected but not sure how to modify to /dev/TTUSB0. Thanks!

**Comment by sfeakes** [comment]
You don’t need to edit or change that device file.
/dev/ttyUSB0 is created by the system when you connect a USB device.  The 0 can be any number depending on how many USB devices are connected
